submissions/rubyAdeniregun/php/listview.php

```diff
@@ -25,8 +25,8 @@
                 echo "<td>" . $row['email_address'] . "</td>";
                 echo "<td>" . $row['created_at'] . "</td>";
                 echo "<td>" . $row['actions'] . "</td>";
-                echo "<td><button type=button>View</button>" . $row['actions'] . "</td>";
-            echo "</tr>";
+                echo "<td><button type=button><a href='orderpage.php'>VIEW</a></button>" . $row['actions'] . "</td>";
```

Comment:
Also, the idea is to get all the order details of each customer you have on this page. Say for example you have Adaa as a customer, You are supposed to return details about customer Adaa when I click on the `View` button associated with Adaa